### ISSUE TITLE
[BUG] Scheduler Crash on Reloading Plugins

### ISSUE THREAD

**Issue by ZCK12** [bunkerity/bunkerweb#863]
### What happened?

Uploading a plugin archive as demo'ed in the WebUI video and clicking "RELOAD" leads to a scheduler crash. Specifically I was uploading the Discord plugin as a .tar.gz archive. 

### How to reproduce?

Drag and drop a plugin archive into the drop zone on the plugins page of the webui.
Press RELOAD.
Check the status of the bunkerweb service.

### Configuration file(s) (yaml or .env)

```YAML
Not relevant
```

### Relevant log output

```shell
Jan 14 14:33:14 aegis start.sh[74258]: [2024-01-14 14:33:14] - SCHEDULER - ℹ  - External plugins changed, generating ...
Jan 14 14:33:14 aegis start.sh[74258]: [2024-01-14 14:33:14] - SCHEDULER - ℹ  - Running config saver to save potential ignored external plugins settings ...
Jan 14 14:33:14 aegis start.sh[74258]: [2024-01-14 14:33:14] - SCHEDULER - ❌ - Exception while executing scheduler : Traceback (most recent call last):
Jan 14 14:33:14 aegis start.sh[74258]:   File "/usr/share/bunkerweb/scheduler/main.py", line 581, in <module>
Jan 14 14:33:14 aegis start.sh[74258]:     proc = subprocess_run(
Jan 14 14:33:14 aegis start.sh[74258]:   File "/usr/lib/python3.10/subprocess.py", line 503, in run
Jan 14 14:33:14 aegis start.sh[74258]:     with Popen(*popenargs, **kwargs) as process:
Jan 14 14:33:14 aegis start.sh[74258]:   File "/usr/lib/python3.10/subprocess.py", line 971, in __init__
Jan 14 14:33:14 aegis start.sh[74258]:     self._execute_child(args, executable, preexec_fn, close_fds,
Jan 14 14:33:14 aegis start.sh[74258]:   File "/usr/lib/python3.10/subprocess.py", line 1863, in _execute_child
Jan 14 14:33:14 aegis start.sh[74258]:     raise child_exception_type(errno_num, err_msg, err_filename)
Jan 14 14:33:14 aegis start.sh[74258]: FileNotFoundError: [Errno 2] No such file or directory: 'python'
Jan 14 14:33:14 aegis start.sh[74158]: [2024-01-14 14:33:14] - SYSTEMCTL - ❌ - Scheduler failed
Jan 14 14:33:14 aegis systemd[1]: bunkerweb.service: Main process exited, code=exited, status=1/FAILURE
Jan 14 14:33:15 aegis systemd[1]: bunkerweb.service: Failed with result 'exit-code'.
```

### BunkerWeb version

1.5.5

### What integration are you using?

Linux

### Linux distribution (if applicable)

Ubuntu LTS

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TheophileDiot** [comment]
Hi @ZCK12, thank you for opening this issue, nice finding by the way ! The next version will only use the `python3` executable.

As a work around for now you can install this package:
```shell
sudo apt install python-is-python3
```

[source](https://askubuntu.com/a/321000)

**Comment by fl0ppy-d1sk** [comment]
Hello @ZCK12,

Fix is now available in the v1.5.6 release. Thanks for your feedback !
